### ISSUE TITLE
Session.GetObjectFromHref() does not work for multivalue href's ... is there an alternative method to locate multivalve objects from the href multivalue number?

### ISSUE THREAD

**Issue by slieberth** [OpenIxia/ixnetwork_restpy#44]
hi team,

in my tests, Session.GetObjectFromHref() returns None for multivalue hrefs. 

This is due to the fact that the href for those derives directly under IxNetwork, e.g.
/api/v1/sessions/8020/ixnetwork/multivalue/2

however IxNetwork has not attribute "Multivalue" 
so the search fails  (using own debug prints):

node Ixnetwork[0]: /api/v1/sessions/8020/ixnetwork/ piece multivalue
'Ixnetwork' object has no attribute 'Multivalue'

```
    def GetObjectFromHref(self, href):
        """Given an href get the corresponding object
        
        References are used to establish relationships between disparate nodes.
        This allows a user to get an object using a reference from a node.
        For example:
            A /vport node has a ConnectedTo property that is a reference to an /availableHardware/chassis/card/port node.
            A user can get a vport and use this method to get the port node in order to clear ownership
        
        Args
        ----
        - href (str): A valid href reference
        Returns
        -------
            obj: If the reference is valid an ixnetwork_restpy object or None if the reference cannot be resolved
        """
        try:
            node = self
            for piece in href[href.find('ixnetwork'):].split('/'):
                if piece.isdigit():
                    for item in node:
                        if item.href.split('/').pop() == piece:
                            node = node._read(item.href)
                            break
                else:
                    class_name = '%s%s' % (piece[0].upper(), piece[1:])
                    node = getattr(node, class_name)
                    if hasattr(node, 'find') is True:
                        node.find()
            return node
        except:
            return None
```

is there an alternative way to locate multivalue objects from its href number .... ?

best regards Stefan

**Comment by ajbalogh** [comment]
A fix for this issue will be available in version 1.0.63

**Comment by slieberth** [comment]
Many Thanks ...  
Stefan
